cwel/component/button/button.es

```diff
@@ -0,0 +1,15 @@
+angular.module('cwel').directive('hoverEffect', () => {
```

Comment:
@NathanRodger Do we need this effect on anything but buttons? If so i think this is a case for a 'class' directive matching the `button` class. Which will reduce the attributes